ui/tests/helpers/clients/client-count-helpers.js

```diff
@@ -646,10 +844,369 @@ export const SERIALIZED_ACTIVITY_RESPONSE = {
+            secret_syncs: 100,
+            mounts: [
+              {
+                label: 'pki-engine-0',
+                acme_clients: 100,
+                clients: 100,
+                entity_clients: 0,
+                non_entity_clients: 0,
+                secret_syncs: 0,
+              },
+              {
+                label: 'auth/authid/0',
+                acme_clients: 0,
+                clients: 100,
+                entity_clients: 100,
+                non_entity_clients: 100,
+                secret_syncs: 0,
+              },
+              {
+                label: 'kvv2-engine-0',
+                acme_clients: 0,
+                clients: 100,
+                entity_clients: 0,
+                non_entity_clients: 0,
+                secret_syncs: 100,
+              },
+            ],
+          },
+          mounts_by_key: {
+            'pki-engine-0': {
+              label: 'pki-engine-0',
+              acme_clients: 100,
+              clients: 100,
+              entity_clients: 0,
+              non_entity_clients: 0,
+              secret_syncs: 0,
+              timestamp: '2023-07-01T00:00:00Z',
+              month: '7/23',
+              new_clients: {
+                month: '7/23',
+                timestamp: '2023-07-01T00:00:00Z',
+                label: 'pki-engine-0',
+                acme_clients: 100,
+                clients: 100,
+                entity_clients: 0,
+                non_entity_clients: 0,
+                secret_syncs: 0,
+              },
+            },
+            'auth/authid/0': {
+              label: 'auth/authid/0',
+              acme_clients: 0,
+              clients: 100,
+              entity_clients: 100,
+              non_entity_clients: 100,
+              secret_syncs: 0,
+              timestamp: '2023-07-01T00:00:00Z',
+              month: '7/23',
+              new_clients: {
+                month: '7/23',
+                timestamp: '2023-07-01T00:00:00Z',
+                label: 'auth/authid/0',
+                acme_clients: 0,
+                clients: 100,
+                entity_clients: 100,
+                non_entity_clients: 100,
+                secret_syncs: 0,
+              },
+            },
+            'kvv2-engine-0': {
+              label: 'kvv2-engine-0',
+              acme_clients: 0,
+              clients: 100,
+              entity_clients: 0,
+              non_entity_clients: 0,
+              secret_syncs: 100,
+              timestamp: '2023-07-01T00:00:00Z',
+              month: '7/23',
+              new_clients: {
+                month: '7/23',
+                timestamp: '2023-07-01T00:00:00Z',
+                label: 'kvv2-engine-0',
+                acme_clients: 0,
+                clients: 100,
+                entity_clients: 0,
+                non_entity_clients: 0,
+                secret_syncs: 100,
+              },
+            },
+          },
+        },
+      },
+      new_clients: {
```

Comment:
This is supposed to be the month with no new clients, right? So this isn't how I'd expect the `new_clients` key to look after it's been serialized.

Reply:
No, the months are as follows: 
- 6/23 - no clients
- 7/23 - new clients
- 8/23 - clients, no new clients
- 9/23 - new clients (original values)